operator/auth/cell.go

```diff
@@ -11,23 +11,23 @@ import (
 )
 
 const (
-	MTLSEnabled = "mesh-auth-mtls-enabled"
+	mutualAuthEnabled = "mesh-mutual-auth-enabled"
```

Comment:
This became magically unexported, do we export these in other packages too?

Reply:
yeah, I noticed that, the compilation didn't fail, so I think it's not imported in other package right now.

Reply:
in that case it probably is fine